tests/OrganizationInvitationsTest.php

```diff
@@ -14,12 +14,7 @@ public function setUp()
     {
         parent::setUp();
 
-        $this->organization = $this->client->createOrganization($this->testMaintainerId, [
-            'name' => 'My org',
-        ]);
-
-        $this->client->addUserToOrganization($this->organization['id'], ['email' => 'spam+spam@keboola.com']);
-        $this->client->removeUserFromOrganization($this->organization['id'], $this->superAdmin['id']);
+        $this->client->addUserToMaintainer($this->testMaintainerId, ['email' => 'spam+spam@keboola.com']);
```

Comment:
tohle je aby tam zbyl ten jeden, je to tak?

Reply:
presne tak